src/lib/journey/stages/webauthn.svelte

```diff
@@ -29,12 +34,19 @@
   const formHeaderId = 'genericStepHeader';
   const formElementId = 'genericStepForm';
 
+  let deviceName = ''; 
+  let askedForDeviceName = false;
   let alertNeedsFocus = false;
   let formMessageKey = '';
   let formAriaDescriptor = 'genericStepHeader';
   let formNeedsFocus = false;
   let webAuthnType = FRWebAuthn.getWebAuthnStepType(step);
 
+  const updateDeviceName = (event: Event) => {
+    const target = event.target as unknown as { value: string } 
+    deviceName = target.value;
+  }
```

Comment:
Can we convert this to a named function syntax like the function on line 65, rather than an arrow function? The convention in this project is to use named functions when they stand alone, but arrow functions when they declared "inline" or as an object method.